Source/JNA/waffle-jna/src/main/java/waffle/util/AuthorizationHeader.java

```diff
@@ -149,10 +149,11 @@ public boolean isSPNegTokenInitMessage() {
      * request with the POSTed data. This is to avoid the situation where user's credentials might be potentially
      * invalid, and all this data is being POSTed across the wire.
      *
-     * @return True if request is an NTLM POST or PUT with an Authorization header and no data.
+     * @return True if request is an NTLM POST, PUT, or DELETE with an Authorization header and no data.
      */
     public boolean isNtlmType1PostAuthorizationHeader() {
-        if (!"POST".equals(this.request.getMethod()) && !"PUT".equals(this.request.getMethod())) {
+        if (!"POST".equals(this.request.getMethod()) && !"PUT".equals(this.request.getMethod())
+                && !"DELETE".equals(this.request.getMethod())) {
```

Comment:
AFAIK the NTLM spec says these two methods must be POST or PUT and that the browser will only send those two preemptively - where/when has this changed and with what browser are we seeing it?

Reply:
Issue was stated to be with IE.  There was a link on the original PR, possibly on here still as well.

Reply:
I'm now not convinced this was fixing anything.  The two links I read entirely through and it's regarding a POST without a post body during pre-authentication.  That is quite different situation than a DELETE being used.  I'll look to remove this unless there is compelling reason for it to stay.  @MoreHeapSpace Can you elaberate as to why DELETE solves an issue here?

Reply:
Sorry for the late reply. I've encountered this problem during pre-authentication on a DELETE action that contains a body. Yes the articles do only document about POST, sorry should have clarified that the same process happens with the DELETE action

Reply:
If the spec doesn't call for DELETE we should be very careful adding it. What's the client that's exhibiting this behavior?

Reply:
Agreed, and the client is IE 11

Reply:
I suspect the same will happen the PATCH method. In fact with any request that contains a body. I have yet to test these as my rest api doesn't make use of that method.

While the NTLM spec only refers to POST or PUT, I believe IE will lock the body all requests